ddtrace/llmobs/_integrations/langgraph.py

```diff
@@ -107,7 +107,11 @@ def _handle_finished_graph(self, graph_span, finished_tasks, is_subgraph_node):
     def _link_task_to_parent(self, task_id, task, finished_task_names_to_ids):
         """Create the span links for a queued task from its triggering parent tasks."""
         task_config = getattr(task, "config", {})
-        task_triggers = task_config.get("metadata", {}).get("langgraph_triggers", [])
+        task_triggers = _normalize_triggers(
```

Comment:
Is this related to langchain-langgraph interactions? Or is this a separate fix rolled into this PR?

Reply:
specific fix for langgraph. good callouts on these last two, let me highlight them separately in the PR description/what they're doing! (in this case i might've missed a docstring for this `_normalize_triggers` method too)